openmrs/src/main/java/org/motechproject/openmrs/domain/Encounter.java

```diff
@@ -135,6 +136,63 @@ public boolean equals(Object obj) {
         return Objects.equals(this.uuid, other.uuid) && Objects.equals(this.display, other.display) &&
                 Objects.equals(this.location, other.location) && Objects.equals(this.encounterType, other.encounterType) &&
                 Objects.equals(this.encounterDatetime, other.encounterDatetime) && Objects.equals(this.patient, other.patient) &&
-                Objects.equals(this.provider, other.provider) && Objects.equals(this.obs, other.obs);
+                Objects.equals(this.encounterProviders, other.encounterProviders) && Objects.equals(this.obs, other.obs);
+    }
+
```

Comment:
add javadoc like in [Observation.java](https://github.com/motech/modules/blob/master/openmrs/src/main/java/org/motechproject/openmrs/domain/Observation.java)